server/build.gradle

```diff
@@ -33,6 +34,14 @@ configurations {
     publicResources
 }
 
+flyway {
```

Comment:
I like this better in the application.yml. 

This isn't even correct for test for example which will be
datasources:
  default:
    url:  jdbc:tc:postgresql:///
    driverClassName: org.testcontainers.jdbc.ContainerDatabaseDriver